Program.cs

```diff
@@ -19,13 +19,23 @@ public static List<string> ReadWordsFromFile(string filePath)
         return words;
     }
 
+    public static List<string> RemoveDuplicateWords(List<string> words)
+    {
+        HashSet<string> uniqueWords = new HashSet<string>(words);
+        return new List<string>(uniqueWords);
+    }
+
     public static void Main()
     {
         List<string> wordsList = ReadWordsFromFile("C:\\Users\\Zoran\\Desktop\\Dario_Internship\\#3_Words\\text.txt");
```

Comment:
We could use a _relative file path_ here, considering that the `text.txt` file is located inside of the repository as well.